### ISSUE TITLE
Stan Addon - Can't login after password changed

### ISSUE THREAD

**Issue by drelephant** [matthuisman/slyguy.addons#881]
I just changed my Stan password, and I can login on the web.

Now when I go to the Kodi Stan Addon (v0.5.1), I get a message:
```
Stan - Error
Failed to login / refresh token.
Message: Streamco.Login.AccountDetailsChanged
```
But I can't find a way to reset the token. I get that message when I try to go into the Addon, and even Uninstalling/Reinstalling the addon (and choosing to delete all data for the addon) doesn't work.

How can I clear the token?

**Comment by drelephant** [comment]
Even uninstalling the entire Slyguy repository and all addons doesn't clear it!

**Comment by matthuisman** [comment]
There isn't anything to clear. That error is coming from their server

**Comment by drelephant** [comment]
Yes. It came up with a wrong password thing, then I could logout/login with the new password.

**Comment by drelephant** [comment]
I actually have kodi installed on both my Windows PC and my Ubuntu HTPC, and both of them are doing it.

I don't really understand how there can be no data to clear.

Surely there must be a place that the login is saved??

How can I get it to re-request the login, when you go to stan.com.au/authorize or whatever it was and enter the code number to save the token for kodi?

**Comment by matthuisman** [comment]
Oh, Have you clicked logout? Or does it not load the main screen?

**Comment by drelephant** [comment]
It doesn't load the main screen, the only option you get is "OK" at that message box:
```
Stan - Error
Failed to login / refresh token.
Message: Streamco.Login.AccountDetailsChanged
```

**Comment by matthuisman** [comment]
Oh right. I'll need to sort that. For now, rename / delete the .db file in
user data addon days slyguy common

**Comment by drelephant** [comment]
Sorry, what's the exact name of the db file?

**Comment by drelephant** [comment]
Nevermind, found it. `%appdata%\Kodi\userdata\addon_data\script.module.slyguy\settings.db`

Thanks again!

**Comment by matthuisman** [comment]
ill do a few things
1) add a "clear data" / reset addon to the settings for the addon
2) try/except around the initial error so the logout / main menu is still shown

**Comment by matthuisman** [comment]
I have now implemented #1
Addon settings -> addon name -> Reset Add-on

That will wipe all settings / userdata / bookmarks / search for the addon.

If you cant get into the add-on itself due to error - there is always the backup way to settings via
Slyguy Common -> Stan -> Stan -> Reset Add-on
(This route doesnt go via the main menu but directly to the settings for each addon)

**Comment by matthuisman** [comment]
#2 now implemented in [plugin.video.stan.au -> v0.5.2](https://github.com/matthuisman/slyguy.addons/commit/20ae760d8a3e3d6a2c6576964ef64de10b0cdd44) so if the menu nav fails to download, itll show an error but then show the rest of the menu so can logout etc

With both of these - no one should get stuck again.

Thanks for bringing this issue up
